[
  {
    "url": "https://api.github.com/repos/quarnster/SublimeClang/issues/comments/10139177",
    "html_url": "https://github.com/quarnster/SublimeClang/issues/162#issuecomment-10139177",
    "issue_url": "https://api.github.com/repos/quarnster/SublimeClang/issues/162",
    "id": 10139177,
    "user": {
      "login": "quarnster",
      "id": 1230389,
      "avatar_url": "https://secure.gravatar.com/avatar/10a89302c6736c988ce06d8a41c923c0?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png",
      "gravatar_id": "10a89302c6736c988ce06d8a41c923c0",
      "url": "https://api.github.com/users/quarnster",
      "html_url": "https://github.com/quarnster",
      "followers_url": "https://api.github.com/users/quarnster/followers",
      "following_url": "https://api.github.com/users/quarnster/following",
      "gists_url": "https://api.github.com/users/quarnster/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/quarnster/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/quarnster/subscriptions",
      "organizations_url": "https://api.github.com/users/quarnster/orgs",
      "repos_url": "https://api.github.com/users/quarnster/repos",
      "events_url": "https://api.github.com/users/quarnster/events{/privacy}",
      "received_events_url": "https://api.github.com/users/quarnster/received_events",
      "type": "User"
    },
    "created_at": "2012-11-07T06:36:42Z",
    "updated_at": "2012-11-07T06:36:42Z",
    "body": "I've never seen something like that. Try putting \"debug_options\": true in your user settings and keep an eye in the python console when the issue happens to see what compilation options are used to parse the translation unit.\r\n\r\nUnless I get better reproduction steps than \"it happens randomly\" I'll not investigate this."
  },
  {
    "url": "https://api.github.com/repos/quarnster/SublimeClang/issues/comments/10139277",
    "html_url": "https://github.com/quarnster/SublimeClang/issues/162#issuecomment-10139277",
    "issue_url": "https://api.github.com/repos/quarnster/SublimeClang/issues/162",
    "id": 10139277,
    "user": {
      "login": "quarnster",
      "id": 1230389,
      "avatar_url": "https://secure.gravatar.com/avatar/10a89302c6736c988ce06d8a41c923c0?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png",
      "gravatar_id": "10a89302c6736c988ce06d8a41c923c0",
      "url": "https://api.github.com/users/quarnster",
      "html_url": "https://github.com/quarnster",
      "followers_url": "https://api.github.com/users/quarnster/followers",
      "following_url": "https://api.github.com/users/quarnster/following",
      "gists_url": "https://api.github.com/users/quarnster/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/quarnster/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/quarnster/subscriptions",
      "organizations_url": "https://api.github.com/users/quarnster/orgs",
      "repos_url": "https://api.github.com/users/quarnster/repos",
      "events_url": "https://api.github.com/users/quarnster/events{/privacy}",
      "received_events_url": "https://api.github.com/users/quarnster/received_events",
      "type": "User"
    },
    "created_at": "2012-11-07T06:45:29Z",
    "updated_at": "2012-11-07T06:45:29Z",
    "body": "Actually, I think I know what it is and have a very reproducible test. Looking at finding a solution now."
  },
  {
    "url": "https://api.github.com/repos/quarnster/SublimeClang/issues/comments/10139307",
    "html_url": "https://github.com/quarnster/SublimeClang/issues/162#issuecomment-10139307",
    "issue_url": "https://api.github.com/repos/quarnster/SublimeClang/issues/162",
    "id": 10139307,
    "user": {
      "login": "JustSid",
      "id": 301144,
      "avatar_url": "https://secure.gravatar.com/avatar/625bc443534e86742d2f865a4f1a5ad7?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png",
      "gravatar_id": "625bc443534e86742d2f865a4f1a5ad7",
      "url": "https://api.github.com/users/JustSid",
      "html_url": "https://github.com/JustSid",
      "followers_url": "https://api.github.com/users/JustSid/followers",
      "following_url": "https://api.github.com/users/JustSid/following",
      "gists_url": "https://api.github.com/users/JustSid/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/JustSid/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/JustSid/subscriptions",
      "organizations_url": "https://api.github.com/users/JustSid/orgs",
      "repos_url": "https://api.github.com/users/JustSid/repos",
      "events_url": "https://api.github.com/users/JustSid/events{/privacy}",
      "received_events_url": "https://api.github.com/users/JustSid/received_events",
      "type": "User"
    },
    "created_at": "2012-11-07T06:47:23Z",
    "updated_at": "2012-11-07T06:48:58Z",
    "body": "Wish I could give you better steps to reproduce. However, upon enabling `debug_options`, it seems like a problem with expanding the include paths (both files are in the same directory):\r\n\r\n    Will compile file /Users/Sidney/Projects/Active/Firedrake/sys/container/array.c with the following options:\r\n    ['-I/Users/Sidney/Library/Application Support/Sublime Text 2/Packages/SublimeClang/clang/include', u'-I/sys/.', u'-I/bin/.', u'-I/lib/.', u'-m32', u'-Wall', u'-Wextra', u'-std=c99', u'-nostdlib', u'-nostdinc', u'-fno-builtin', u'-pedantic-errors', '-x', u'c']\r\n    Will compile file /Users/Sidney/Projects/Active/Firedrake/sys/container/atree.c with the following options:\r\n    ['-I/Users/Sidney/Library/Application Support/Sublime Text 2/Packages/SublimeClang/clang/include', u'-I/Users/Sidney/Projects/Active/Firedrake/sys/.', u'-I/Users/Sidney/Projects/Active/Firedrake/bin/.', u'-I/Users/Sidney/Projects/Active/Firedrake/lib/.', u'-m32', u'-Wall', u'-Wextra', u'-std=c99', u'-nostdlib', u'-nostdinc', u'-fno-builtin', u'-pedantic-errors', '-x', u'c']\r\n\r\nHere is the relevant part of the project for the include paths (inside the `sublimeclang_options` array):\r\n\r\n    \"-I${folder:${project_path:Firedrake.sublime-project}}/sys/.\",\r\n    \"-I${folder:${project_path:Firedrake.sublime-project}}/bin/.\",\r\n    \"-I${folder:${project_path:Firedrake.sublime-project}}/lib/.\",\r\n\r\nEdit: Whoops, didn't see your last comment. Hope you can find a fix for this! :)"
  },
  {
    "url": "https://api.github.com/repos/quarnster/SublimeClang/issues/comments/10139539",
    "html_url": "https://github.com/quarnster/SublimeClang/issues/162#issuecomment-10139539",
    "issue_url": "https://api.github.com/repos/quarnster/SublimeClang/issues/162",
    "id": 10139539,
    "user": {
      "login": "quarnster",
      "id": 1230389,
      "avatar_url": "https://secure.gravatar.com/avatar/10a89302c6736c988ce06d8a41c923c0?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png",
      "gravatar_id": "10a89302c6736c988ce06d8a41c923c0",
      "url": "https://api.github.com/users/quarnster",
      "html_url": "https://github.com/quarnster",
      "followers_url": "https://api.github.com/users/quarnster/followers",
      "following_url": "https://api.github.com/users/quarnster/following",
      "gists_url": "https://api.github.com/users/quarnster/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/quarnster/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/quarnster/subscriptions",
      "organizations_url": "https://api.github.com/users/quarnster/orgs",
      "repos_url": "https://api.github.com/users/quarnster/repos",
      "events_url": "https://api.github.com/users/quarnster/events{/privacy}",
      "received_events_url": "https://api.github.com/users/quarnster/received_events",
      "type": "User"
    },
    "created_at": "2012-11-07T07:04:48Z",
    "updated_at": "2012-11-07T07:04:48Z",
    "body": "@JustSid, could you try to apply e71a631da60b886a6c985f6fe7e843703140b48a to your local copy and see if that fixes the issue for you?"
  },
  {
    "url": "https://api.github.com/repos/quarnster/SublimeClang/issues/comments/10139811",
    "html_url": "https://github.com/quarnster/SublimeClang/issues/162#issuecomment-10139811",
    "issue_url": "https://api.github.com/repos/quarnster/SublimeClang/issues/162",
    "id": 10139811,
    "user": {
      "login": "quarnster",
      "id": 1230389,
      "avatar_url": "https://secure.gravatar.com/avatar/10a89302c6736c988ce06d8a41c923c0?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png",
      "gravatar_id": "10a89302c6736c988ce06d8a41c923c0",
      "url": "https://api.github.com/users/quarnster",
      "html_url": "https://github.com/quarnster",
      "followers_url": "https://api.github.com/users/quarnster/followers",
      "following_url": "https://api.github.com/users/quarnster/following",
      "gists_url": "https://api.github.com/users/quarnster/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/quarnster/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/quarnster/subscriptions",
      "organizations_url": "https://api.github.com/users/quarnster/orgs",
      "repos_url": "https://api.github.com/users/quarnster/repos",
      "events_url": "https://api.github.com/users/quarnster/events{/privacy}",
      "received_events_url": "https://api.github.com/users/quarnster/received_events",
      "type": "User"
    },
    "created_at": "2012-11-07T07:22:54Z",
    "updated_at": "2012-11-07T07:22:54Z",
    "body": "Added another commit that might be needed."
  },
  {
    "url": "https://api.github.com/repos/quarnster/SublimeClang/issues/comments/10139886",
    "html_url": "https://github.com/quarnster/SublimeClang/issues/162#issuecomment-10139886",
    "issue_url": "https://api.github.com/repos/quarnster/SublimeClang/issues/162",
    "id": 10139886,
    "user": {
      "login": "JustSid",
      "id": 301144,
      "avatar_url": "https://secure.gravatar.com/avatar/625bc443534e86742d2f865a4f1a5ad7?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png",
      "gravatar_id": "625bc443534e86742d2f865a4f1a5ad7",
      "url": "https://api.github.com/users/JustSid",
      "html_url": "https://github.com/JustSid",
      "followers_url": "https://api.github.com/users/JustSid/followers",
      "following_url": "https://api.github.com/users/JustSid/following",
      "gists_url": "https://api.github.com/users/JustSid/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/JustSid/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/JustSid/subscriptions",
      "organizations_url": "https://api.github.com/users/JustSid/orgs",
      "repos_url": "https://api.github.com/users/JustSid/repos",
      "events_url": "https://api.github.com/users/JustSid/events{/privacy}",
      "received_events_url": "https://api.github.com/users/JustSid/received_events",
      "type": "User"
    },
    "created_at": "2012-11-07T07:27:31Z",
    "updated_at": "2012-11-07T07:27:31Z",
    "body": "Great, this seems to indeed have fixed the problem! (Gave the first commit a try of ~15 minutes, which before was more than enough time to see this problem). Thanks a lot for the quick help and fix!"
  }
]
